modules/uhsg_feedback/uhsg_feedback.module

```diff
@@ -43,11 +43,11 @@ function uhsg_feedback_mail_alter(&$message) {
 
     // I would like to get a response to my feedback.
     if ($contact_message->get('field_feedback_respond')->value) {
-      $message['body'][] = t('I would like to get a response to my feedback') . ': ' . $replyTo;
+      $message['body'][] = t('I would like to get a response to my feedback: @replyTo', ['@replyto' => $replyTo]);
```

Comment:
Translated string has been changed. Have the translations been updated to reflect this change?